src/Rings/mpoly-localizations.jl

```diff
@@ -1313,7 +1324,7 @@ Ideals in localizations of polynomial rings.
   W::MPolyLocalizedRing{BRT, BRET, RT, RET, MST} 
   
   # Fields for caching
-  groebner_bases::Dict{Symbol, LocalizedBiPolyArray{BRT, BRET, RT, RET, MST}}
+  groebner_bases::Dict{MonomialOrdering, LocalizedBiPolyArray{BRT, BRET, RT, RET, MST}}
```

Comment:
@wbhart : I recall that the hashing of orderings was something that was pending for some time. Is that reliable now? I.e.: Is it safe to use orderings in dictionaries?

Reply:
At least it is done like this in the other `groebner_basis` functions...

Reply:
Then, I guess it's fine by now.